### PR TITLE
Docs/v3 sample notebooks readme and updates

### PR DESCRIPTION
**Pull request by emmalu**
## Summary:
**What** did I do?
- Added README.md file to the `docs/` directory
- Updated raster sample notebook to help address inconsistent behavior

**Why** *** did I do it? (What was the motivation?)
- To help orient users on how/when to use those sample notebooks
- To ensure the sample code works for multiple rasters

**How** *** can someone test or demo my changes?
- Run the code in either MAAP or VEDA environment!

## Checklist before requesting a review:
- [X] My code follows the guidelines of this project
- [X] I  performed a self-review of my code changes
- [ ] I have added my changes to the change log (`HISTORY.rst`)
- [X] I have completed spell-checking, removed unnecessary print statements & commented-out code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by sandrahoang686 — APPROVED** [review]
lgtm

File: docs/READEME.md
Comment (chuckwondo) [inline docs/READEME.md:1]: Check the spelling of this file name. You seem to have one too many `E`s in it.

Comment (sandrahoang686) [reply docs/READEME.md:1]: totally did not catch that, thanks @chuckwondo

Comment (emmalu) [reply docs/READEME.md:1]: Good eyes, @chuckwondo!

**Pull request merged**